pubspec.yaml

```diff
@@ -9,12 +9,12 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  intl: ^0.17.0
+  intl: ^0.17.0-nullsafety.2
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  path: ^1.6.2
+  path: ^1.8.0-nullsafety.3
```

Comment:
```suggestion
  path: ^1.8.0
```

Reply:
Using path 1.8.0 gives the pub error:

```
Because flutter_localized_locales depends on flutter_test any from sdk which depends on path 1.8.0-nullsafety.3, path 1.8.0-nullsafety.3 is required.
```